src/main/java/edu/hm/hafner/grading/Baseline.java

```diff
@@ -0,0 +1,16 @@
+package edu.hm.hafner.grading;
+
+public enum Baseline {
+    PROJECT,
+    MODIFIED_FILES,
+    MODIFIED_LINES;
+
+    public static Baseline fromString(String value) {
+        for (Baseline baseline : values()) {
+            if (baseline.name().equalsIgnoreCase(value)) {
+                return baseline;
+            }
+        }
+        return PROJECT;
```

Comment:
It makes more sense to throw an exception so users directly see that the invocation parameters are broken (fail fast).